package.json

```diff
@@ -845,13 +853,20 @@
     "dependencies": {
         "antlr4ts": "^0.4.1-alpha.0",
         "azure-arm-cosmosdb": "^1.1.2",
+        "azure-arm-postgresql": "^4.3.0",
+        "azure-arm-resource": "^3.0.0-preview",
         "bson": "^1.1.3",
         "documentdb": "^1.14.2",
         "fs-extra": "^4.0.2",
         "mongodb": "^3.3.2",
         "mongodb-extended-json": "^1.10.0",
         "ms-rest": "^2.2.1",
         "ms-rest-azure": "^2.3.1",
+        "node-uuid": "1.4.8",
+        "pg": "^7.18.1",
+        "pg-structure": "^5.10.7",
+        "socket.io": "^1.7.3",
```

Comment:
Some of these were removed in https://github.com/microsoft/vscode-cosmosdb/pull/1362. Make sure the diff here only shows the postgres packages you need